db/migrate/20160502224706_create_products.rb

```diff
@@ -0,0 +1,15 @@
+class CreateProducts < ActiveRecord::Migration
+  def change
+    create_table :products do |t|
+      t.string :name, null: false
+      t.integer :price, null: false
+      # user_id here meaning merchant_id
```

Comment:
Here's a link for how you can implement a merchant as a specific type of user for certain circumstances. It is an example of how someone did a similar thing only with the concept of a user vs an author.

http://stackoverflow.com/questions/30694403/how-to-assign-post-to-user-through-user-id-and-author-id